src/__tests__/components/__snapshots__/FileUpload.itest.js.snap

```diff
@@ -1,629 +1,3 @@
-                    "className": "file-icon",
-                    "fixedWidth": false,
-                    "flip": null,
-                    "icon": "file",
-                    "inverse": false,
-                    "listItem": false,
-                    "mask": null,
-                    "pull": null,
-                    "pulse": false,
-                    "rotation": null,
-                    "size": null,
-                    "spin": false,
-                    "symbol": false,
-                    "title": "",
-                    "transform": null,
-                  },
-                  "ref": null,
-                  "rendered": null,
-                  "type": [Function],
-                },
-                Object {
-                  "instance": null,
-                  "key": undefined,
-                  "nodeType": "host",
-                  "props": Object {
-                    "children": "file1",
-                    "className": "file-name",
-                  },
-                  "ref": null,
-                  "rendered": "file1",
-                  "type": "span",
-                },
-                Object {
-                  "instance": null,
-                  "key": undefined,
-                  "nodeType": "host",
-                  "props": Object {
-                    "children": <div>
-                        
-                      0
-                      Mb
-                    </div>,
-                    "className": "file-size",
-                  },
-                  "ref": null,
-                  "rendered": Object {
-                    "instance": null,
-                    "key": undefined,
-                    "nodeType": "host",
-                    "props": Object {
-                      "children": Array [
-                        "  ",
-                        0,
-                        "Mb",
-                      ],
-                    },
-                    "ref": null,
-                    "rendered": Array [
-                      "  ",
-                      0,
-                      "Mb",
-                    ],
-                    "type": "div",
-                  },
-                  "type": "span",
-                },
-              ],
-              "type": "li",
-            },
-          ],
-          "type": "ul",
-        },
-      ],
-      "type": "div",
-    },
-  ],
-  Symbol(enzyme.__options__): Object {
-    "adapter": ReactSixteenAdapter {
-      "options": Object {
-        "enableComponentDidUpdateOnSetState": true,
-        "lifecycles": Object {
-          "componentDidUpdate": Object {
-            "onSetState": true,
-          },
-          "getDerivedStateFromProps": true,
-          "getSnapshotBeforeUpdate": true,
-          "setState": Object {
-            "skipsComponentDidUpdateOnNullish": true,
-          },
-        },
-      },
-    },
-  },
-}
-`;
+exports[`FileUpload Component matches snapshot 1`] = `ShallowWrapper {}`;
```

Comment:
looks like there's sth wrong with the shallow rendering

Reply:
Fixed. In the latest version of Jest you need to add a JSON serializer. I added the recommended `enzyme-to-json` and it fixed all the issues.